libr/search/search.c

```diff
@@ -427,7 +348,7 @@ R_API RList *r_search_find(RSearch *s, ut64 addr, const ut8 *buf, int len) {
 
 /* --- keywords --- */
 R_API int r_search_kw_add(RSearch *s, RSearchKeyword *kw) {
-	if (!kw) return false;
+	if (!kw->keyword_length) return false;
```

Comment:
IMHO using curly braces everywhere does not enhance readability, especially for short conditional statements such as:
```c
if (!kw->keyword_length) {
  return false;
}
```

Linux coding style allows:
```c
        if (!buffer)
                return -ENOMEM;
```

Google C++ Style Guide (https://google.github.io/styleguide/cppguide.html#Conditionals) allows the statement put on the same line if it enhances readability:

```c
if (x == kFoo) return new Foo();
```